plugin-server/src/config/config.ts

```diff
@@ -71,7 +71,7 @@ export function getDefaultConfig(): PluginsServerConfig {
         JOB_QUEUE_S3_AWS_REGION: 'us-west-1',
         JOB_QUEUE_S3_BUCKET_NAME: '',
         JOB_QUEUE_S3_PREFIX: '',
-        CRASH_IF_NO_PERSISTENT_JOB_QUEUE: false,
+        CRASH_IF_NO_PERSISTENT_JOB_QUEUE: true,
```

Comment:
Convince me on the new default. What has changed between then and now that crashing the plugin-server is OK?

Reply:
See the description:

> We've had this setting on cloud for months, I don't see why we wouldn't want it on self-hosted, flipping the default, so self-hosted folks are more similar to cloud.

Reply:
But cloud has this as False not True.

Reply:
so what has changed is that we've tested that this setting is a good one for long enough on cloud. I suspect this could have been causing plugin-server to get stuck in a bad state on self-hosted & we've asked people manually to restart as a remediation.

Reply:
🤦‍♀️